### PR TITLE
CLOUDP-92631: mongodb log to stdout

### LINKED ISSUE #570: Log takes all space and mongodb is down
**What did you do to encounter the bug?**
We are running a cluster with 3 nodes, it works about 2 months.
Once one replica gone down.
After some investigation we found that the file /var/log/mongodb-mms-automation/mongodb.log takes all spaces in /var/log/mongodb-mms-automation mount.

**What did you expect?**
I expect /var/log/mongodb-mms-automation stores something for agent or similar but not mongod logs.
I expect mongod writes logs into stdout as containerized applications typically behave.

**What happened instead?**
No space left in /var/log/mongodb-mms-automation, mongod and agent works improperly.

**Operator Information**
 - Operator Version: 0.6.0
 - MongoDB Image used: registry.hub.docker.com/library/mongo:4.4.4

**Kubernetes Cluster Information**
 - Distribution: GKE
 - Version: 1.18
 - Image Registry location (quay, or an internal registry): default operator configuration

### PR DESCRIPTION
**Pull request by bznein**
closes #570 
by not defaulting to saving `mongodb` logs to file. Users can still obtain the old behaviour through `spec.additionalMongodConfig`

### All Submissions:

* [ ] Have you opened an Issue before filing this PR?
* [ ] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?
* [ ] Have you signed all of your commits?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [ ] Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: pkg/automationconfig/automation_config_builder.go [hunk 1]
```diff
@@ -214,11 +213,6 @@ func (b *Builder) Build() (AutomationConfig, error) {
 			Version:                     b.mongodbVersion,
 			AuthSchemaVersion:           5,
 		}
-		process.SetSystemLog(SystemLog{
-			Destination: "file",
-			Path:        path.Join(DefaultAgentLogPath, "/mongodb.log"),
-			LogAppend:   true,
-		})
```
Comment (bznein) [inline pkg/automationconfig/automation_config_builder.go:221]: By removing this `mongodb` now by defaults sends its output to stdout: https://docs.mongodb.com/manual/reference/configuration-options/#mongodb-setting-systemLog.destination

File: pkg/automationconfig/automation_config_secret_test.go [hunk 2]
```diff
@@ -25,7 +25,7 @@ func TestEnsureSecret(t *testing.T) {
 			SetNamespace(secretNsName.Namespace).
```
Comment (bznein) [inline pkg/automationconfig/automation_config_secret_test.go:25]: changes in this file are unrelated and come from pre-commit hook errors

**Review by chatton — APPROVED** [review]
LGTM thanks for the quick fix. Can we update the release notes before merging though!
